solidity/docs/generate-api-docs.js

```diff
@@ -30,39 +30,43 @@ jsonFiles.forEach((file) => {
   let section = "== `" + json.contractName + "`\n\n"
 
   for (const signature in json.userdoc.methods) {
-    const props = json.userdoc.methods[signature]
```

Comment:
We should just squash this with the previous commit as a single “fix lint issues after updating lint config” commit. The description can mention it has both automated and manual fixes if we want.